pkg/scheduler/scheduler.go

```diff
@@ -457,14 +457,17 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 		// will fit due to the preemption. It is also possible that a different pod will schedule
 		// into the resources that were preempted, but this is harmless.
 		nominatedNode := ""
-		if fitError, ok := err.(*core.FitError); ok {
+		if fitError, ok := err.(*framework.FitError); ok {
 			if !fwk.HasPostFilterPlugins() {
 				klog.V(3).InfoS("No PostFilter plugins are registered, so no preemption will be performed")
 			} else {
 				// Run PostFilter plugins to try to make the pod schedulable in a future scheduling cycle.
 				result, status := fwk.RunPostFilterPlugins(ctx, state, pod, fitError.FilteredNodesStatuses)
 				if status.Code() == framework.Error {
 					klog.ErrorS(nil, "Status after running PostFilter plugins for pod", klog.KObj(pod), "status", status)
+				} else if status.IsUnschedulable() {
+					// In case there is no node available after running of post filter plugins, the details could be found in the log.
+					klog.Warningf("Status after running PostFilter plugins for pod", "pod", klog.KObj(pod), "status", status)
```

Comment:
the new logging framework doesn't support warnings (ErrorS and InfoS only), existing warning should be changed to either an error or info, and in this case it is info, so lets revert this change.

Reply:
Good to know!  have reverted.